models/src/main/scala/coop/rchain/models/rholang/sorter/ordering.scala

```diff
@@ -6,12 +6,24 @@ import coop.rchain.models.rholang.sorter.ScoredTerm._
 import monix.eval.Coeval
 import cats.implicits._
 
-import scala.collection.immutable
+import scala.collection.{immutable, GenTraversableOnce}
 
 //FIXME the `.sort` methods in this file should return via F[_] : Sync, and the corresponding ParSet and ParMap should
 //be constructed via factory methods also returning via F. Otherwise we risk StackOverflowErrors.
 object ordering {
 
+  implicit class ParOps(par: Par) {
+    implicit val sync = implicitly[Sync[Coeval]]
+
+    def sort: Par = Sortable[Par].sortMatch[Coeval](par).map(_.term).value()
+  }
+
+  implicit class GenTraversableOnceOps(ps: GenTraversableOnce[Par]) {
+    implicit val sync = implicitly[Sync[Coeval]]
+
+    def sort: GenTraversableOnce[Par] = ps.toList.sort
```

Comment:
Please remove those implicit conversions - I'd like Sortable to be the entry point for sorting for all new usages, as it's explicit about monads being used underneath. At some point we're going to have to join all the computations into a single one, in a single monad - otherwise performance will continue to suffer and there's no stacksafety

Reply:
parOps.sort should be just sort(Par) - a private method in the SortedMap.

GenTraverasableOps is not needed at all - it's enough if we just:
`kvs.foldLeft(sortedMap) { map, kv -> sortedMap - kv }` in the `--` mehtod